package.json

```diff
@@ -113,6 +113,7 @@
     "expo-av": "14.0.7",
     "expo-image": "1.12.15",
     "expo-image-manipulator": "12.0.5",
+    "expo-navigation-bar": "~3.0.7",
```

Comment:
I had to add this library again, because we need to manage navigation bar button style (dark/light).

Since user can customize the theme of app independently to device settings we also have to manage it dynamically from JS code.

I checked and one method that we use (`setButtonStyleAsync`) doesn't use deprecated API. However I remember, that this particular library wasn't accepted in the first round of fixing the nav bar color management, so i'm happy to discuss further steps here 🙌

Reply:
@kirillzyusko I noticed you are adding the library here again even though in Slack we agreed not to use it https://expensify.slack.com/archives/C01GTK53T8Q/p1731618817760309?thread_ts=1731092817.909359&cid=C01GTK53T8Q

Can you please come back to the thread and summarize the problems you are facing with not using the library or explain clearly why we cannot just handle this on our own? Thanks!